<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFilterDialog</class>
 <widget class="QDialog" name="CreateFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>215</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter type</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="noFilterRadio">
          <property name="text">
           <string>No filter</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rangeFilterRadio">
          <property name="text">
           <string>Range</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="listFilterRadio">
          <property name="text">
           <string>List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="rangerFilterGroup">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Range filter</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Min ID: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="minIDSpin">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="minIDCombo"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Max ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxIDSpin">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="maxIDCombo"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="listFilterGroup">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>List filter</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="idSpin">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="idCombo"/>
           </item>
           <item>
            <widget class="QToolButton" name="addToListButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../assets/assets.qrc">
               <normaloff>:/add.png</normaloff>:/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removeFromListButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../assets/assets.qrc">
               <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="idList"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/assets.qrc">
         <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/assets.qrc">
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rangeFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangerFilterGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>listFilterGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
